### PR TITLE
link to better performance of the same talk

### PR DESCRIPTION
**Pull request by ztellman**
This new version is a bit more refined, and has the added benefit of me not being extremely jetlagged when I gave it.

### CODE REVIEW

**Review by hellerve — APPROVED** [review]
Thank you for your update!

Also, let me take this moment to say that you might be the single most influential voice in my struggle to come to terms with what it means to be a technologist. Your talks have helped me discover Borges, Foucault, Kuhn, Baudrilliard, and Kripke, among others, whose books have had a tremendous impact on me.

Your talks have also shaped my talks from being about technical minutiae to thinking about my domain, my craft, and about thought itself. I’m by no means very good at communicating these very basic yet so complicated things, but you motivated me to try.

In short, thank you for all your work. You’ve made me a better thinker.

**Comment by ztellman** [comment]
That means a lot, thank you!
